src/firefly/java/edu/caltech/ipac/table/TableUtil.java

```diff
@@ -120,6 +120,9 @@ public static Format guessFormat(File inf) throws IOException {
                 } else if (line.startsWith("<VOTABLE") ||
                         (line.contains("<?xml") && line.contains("<VOTABLE "))) {
                     return Format.VO_TABLE;
+                } else if (line.startsWith("<uws") ||
+                        (line.contains("<?xml") && line.contains("<uws "))) {
+                    return Format.UWS;
```

Comment:
This is not a good check for a UWS Job.  It's rather complicated to be exact, but the bare minimum, you should look for is `<uws:job `, because `uws` is just a schema that may contains many other elements that is not `job`.  I don't understand what this is.. `line.contains("<?xml")`.

When looking for `<uws:job `, it's good to be aware that padding is allowed.  `   <uws:job   ` is the same as `<uws:job `.
Try something like this...
```java
line.trim().startsWith("<uws:job ")
```